web/modules/custom/ai_screening_project_track/src/Form/ThresholdsForm.php

```diff
@@ -106,18 +106,18 @@ public function validateThresholds(array &$form, FormStateInterface $form_state)
     foreach ($userInput as $key => $value) {
       $thresholdKeyArr = explode('-', $key);
       $thresholds = [
-        Evaluation::UNDECIDED->getAsLowerCase() => $userInput[implode('-', [
-          Evaluation::UNDECIDED->getAsLowerCase(),
+        Evaluation::UNDECIDED->value => $userInput[implode('-', [
```

Comment:
You could use `ProjectTrackTypeHelper::buildThresholdKey` here … And use `…::getThresholdKeys` in line 107 …